boards/arm/nucleo_h723zg/support/openocd.cfg

```diff
@@ -0,0 +1,28 @@
+# STM32H745ZI Nucleo board OpenOCD ST-LLINK V3 configuration
+#
+# Copyright (c) 2020 Alexander Kozhinov
+# AlexanderKozhinov@yandex.com
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source [find interface/stlink-dap.cfg]
```

Comment:
is stlink-dap required? I think that *standard* stlink should work fine.

Reply:
Okay, Changed to `source [find interface/stlink.cfg]`.